server/src/inmantals/server.py

```diff
@@ -109,11 +122,33 @@ def treeify_reverse(iterator):
                 for k, v in groupby(anchormap, lambda x: x[1].file)
             }
 
+            await self.publish_diagnostics(None)
+
+        except CompilerException as e:
+            if ast_export is not None:
+                error: ast_export.Error = e.export()
+                params: Optional[lsp_types.PublishDiagnosticsParams]
+                if error.location is None:
+                    params = None
+                else:
+                    params = lsp_types.PublishDiagnosticsParams(
+                        uri=error.location.uri,
+                        diagnostics=[
+                            lsp_types.Diagnostic(
+                                range=error.location.range,
+                                severity=lsp_types.DiagnosticSeverity.Error,
+                                message=f"{error.type}: {error.message}",
```

Comment:
Is the type a relevant part of the message?

Reply:
I find it useful but that might be because I've got insight in the compiler. It might be better to just stick with the error message.

Reply:
(don't know either, tbh)